### ISSUE TITLE
Add support for Logitech Craft Keyboard Dial

### ISSUE THREAD

**Issue by thexclu** [fcpxhacks/fcpxhacks#1897]
As title, i think Craft keyboard have powerful dial keyboard, im not sure can CommandPost support Logi Craft?

**Comment by latenitefilms** [comment]
@designedbyclu - Sorry for the delayed reply, I missed this.

I've reached out to Logitech about their SDK, but haven't heard back yet.

**Comment by latenitefilms** [comment]
Sadly, I've had no response from Logitech yet. I'm going to try and chase them on Twitter.

**Comment by latenitefilms** [comment]
Note to self:

- [Craft Crown SDK Documentation](https://github.com/Logitech/logi_craft_sdk/blob/master/documentation/Craft_Crown_SDK.md#10-crown-sdk)
- [Craft Crown SDK Samples](https://github.com/Logitech/logi_craft_sdk/tree/master/samples)

**Comment by latenitefilms** [comment]
@designedbyclu - Curious... Do you actually own a Logitech Craft Keyboard?

**Comment by thexclu** [comment]
Yes, i do. i still using it everyday. Wish to make the dial more useful

**Comment by latenitefilms** [comment]
@designedbyclu - I now have a Craft keyboard, and plan to support this in the near future.

**Comment by thexclu** [comment]
That's great, I though the project is not going anymore. Look forward for that, thanks for your effort and time

**Comment by hexeric** [comment]
it would be great to replicate the behaviour of the craft with the official logitech app in photoshop for example. where press+turn lets you switch tools or functions. at the moment you can only set a custom shortcut to the turning and pressing for FCP – press+turn is locked, as is the OSD or UI

**Comment by hexeric** [comment]
yes please!!

**Comment by hexeric** [comment]
any news on this? would be SO cool!

**Comment by latenitefilms** [comment]
It's definitely on the to-do list, as I use a Logitech Craft at home & work. I do have a work-in-progress build currently - I just need to finish and ship it. Hopefully sometime in the next month or so!
